packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.Input/src/InputTypes/ContinuationToken.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.TypeSpec.Generator.Input
+{
+    public class ContinuationToken
```

Comment:
Convention is to preface with Input should we do that here too?

Few other classes too